pkg/manager/internal/controllers/project_controller.go

```diff
@@ -31,10 +31,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/kubermatic/utils/pkg/util"
```

Comment:
imports are sorted weirdly...we have 4 instead of 3 categories.

Reply:
Done, fixed